src/sidebar/components/search-status-bar.js

```diff
@@ -8,6 +8,12 @@ const { withServices } = require('../util/service-context');
 const uiConstants = require('../ui-constants');
 const useStore = require('../store/use-store');
 
+/**
+ * Of the annotations in the thread `annThread`, how many
+ * are currently `visible` in the browser (sidebar)?
+ *
+ * TODO: This function should be a selector or a reusable util
```

Comment:
Frankly I think this function is just fine here. It only needs to be moved out if you want to do so for testing purposes or because you actually need to use it in multiple places.